packages/amplify-util-mock/src/__e2e_v2__/test-synthesizer/transform.ts

```diff
@@ -17,6 +17,7 @@ export const defaultTransformParams: Pick<ExecuteTransformConfig, 'transformersF
     enableAutoIndexQueryNames: true,
     respectPrimaryKeyAttributesOnConnectionField: true,
     enableSearchNodeToNodeEncryption: false,
+    enableTransformerCfnOutputs: true,
```

Comment:
This change is to resolve

![Screenshot 2023-09-27 at 9 54 56 PM](https://github.com/aws-amplify/amplify-cli/assets/11983489/af6ed7f8-6b7a-43dd-8a0c-53a8b89bfe13)

Reply:
@alharris-at did data ship a break ?

Reply:
The break is from catacory-api v3.2.0 and the code is here https://github.com/aws-amplify/amplify-category-api/commit/0c72d1822f8e5ccb3e04a0a49049a459b5fb49e6

Reply:
It turned out that I deleted and regenerated `yarn.lock` when merging `dev` eb3c0bcaff330979bfeab730da8b94455d4a0249